crypto/ec/ecdh_ossl.c

```diff
@@ -66,6 +66,10 @@ int ecdh_simple_compute_key(unsigned char **pout, size_t *poutlen,
     BN_CTX_start(ctx);
     x = BN_CTX_get(ctx);
     y = BN_CTX_get(ctx);
+    if (y == NULL) {
```

Comment:
If you find one BN_CTX that is used in multiple threads (without external synchonization)
you have found one BUG.